### ISSUE TITLE
Picture orientation problem

### ISSUE THREAD

**Issue by Liangdacai** [jiang-du/BlazePose-tensorflow#5]
hello ～I have a question
Data picture, do I need to keep the human body facing up? For example, head down, legs up
thanks

**Comment by jiang-du** [comment]
Good question.
According to the experimental results, I think it is yes.
The results on persons heading up and feet down look good on the validation set. While, my trained blazepose cannot obtain a good result for those persons with strange poses.
Thus, it is a good idea if all the persons are in a similar direction (not exactly the same).
